src/js/config/routes.js

```diff
@@ -67,6 +67,11 @@ module.exports = function (app) {
           })
         .up()
 
+        // Init Contect Page
```

Comment:
Typo in comment. The comment can be removed since it's obvious what's happening here. :)